tools/wpt/commands.json

```diff
@@ -1,6 +1,6 @@
 {
     "run": {"path": "run.py", "script": "run", "parser": "create_parser", "help": "Run tests in a browser",
-            "virtualenv": true, "install": ["requests", "mozinstall", "mozdownload"], "requirements": ["../wptrunner/requirements.txt"]},
```

Comment:
@csnardi removing the install here means `./wpt install firefox browser` fails because they aren't installed?

Reply:
After the second commit, I don't believe so. I didn't test it locally but Travis failed to install Firefox with only the first commit and then was fine after the second.

Reply:
Actually, you're probably right; it still would have failed though prior to this commit I believe.